tests/test_models.py

```diff
@@ -46,3 +46,10 @@ def test_sealed_instance_m2m_access(self):
         message = "Cannot fetch many-to-many field previous_locations on a sealed object."
         with self.assertRaisesMessage(SealedObject, message):
             instance.previous_locations.all()
+
+    def test_sealed_instance_reverse_m2m_access(self):
+        instance = Location.from_db('default', ['latitude', 'longitude'], [1.2, 3.4])
```

Comment:
It looks like the test was failing because deferred instances must always have a pk.